WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -97,11 +99,14 @@ - (XCElementSnapshot *)fb_parentMatchingType:(XCUIElementType)type
   return snapshot;
 }
 
-- (XCElementSnapshot *)fb_parentMatchingOneOfTypes:(NSArray<NSNumber *> *)types
+- (XCElementSnapshot *)fb_findVisibleParentMatchingOneOfTypes:(NSArray<NSNumber *> *)types
 {
   XCElementSnapshot *snapshot = self.parent;
-  while (snapshot && !isSnapshotTypeAmongstGivenTypes(snapshot, types)) {
-      snapshot = snapshot.parent;
+  while (snapshot) {
+    if (isSnapshotTypeAmongstGivenTypes(snapshot, types) && [snapshot isWDVisible] && doesSnapshotHasMoreThanOneVisibleChildrenSnapshots(snapshot)) {
```

Comment:
Thanks @marekcirkos. I know this `fb_findVisibleParentMatchingOneOfTypes` doesnt suggest anything about cell. And thats why I said I will update that in next commit. I just wanted to get your thought on this.

I think underline issue is "just one child is visible and others are not". In this scenario  its failing here https://github.com/facebook/WebDriverAgent/blob/master/WebDriverAgentLib/Categories/XCUIElement%2BFBScrolling.m#L99-L104